backend/manager/modules/bll/src/test/java/org/ovirt/engine/core/bll/scheduling/policyunits/CPUPolicyUnitTest.java

```diff
@@ -58,6 +64,8 @@ public void setUp() {
 
         cluster = new Cluster();
         cluster.setId(Guid.newGuid());
+
+        when(pendingHostResources.pendingHostResources(any(), any())).thenReturn(Collections.emptyList());
```

Comment:
would be nice to have a test with pending resources as well

Reply:
I agree, nice to have. TBH, if we wanted to properly test the class we would end up mocking much of the functionality. I do not see much value in tests like "assert pending resources manager has been called)....